### PR TITLE
finished account check function

### PR DESCRIPTION
**Pull request by threadshare**
finished account check function and add the function of httpapi, convenient to write testcase to call the main chain of some information.
And I have tested the httpapi function, it run well.
Usecase write in httpapi.README.md

### CODE REVIEW

File: eosapi/httpapi/client.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+if __name__ == '__main__':
+    client = Client(['http://eosdapp.oneeos.org:8000'])
+    # res1 = client.get_transaction('c6a6733df77c27094f9f65591f9ce783262b73c2a796aec69ea20545e5dce0e8')
+    # print(res1)
+    # res3 = client.stream_blocks()
+    # for item in res3:
+    #     print(item)
+    # print(res3)
+    # client.get_info()
```
Comment (EOSBIXIN) [inline eosapi/httpapi/client.py]: please remove comments and give a detail description in Readme file

File: eosapi/httpapi/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# client.get_code('zhangzhichao')
+# client.get_block(10)
+# client.get_actions('zhangzhichao',1,10)  # need open filter_on params when you run nodeos
+# client.get_currency_balance('eosio.token','zhangzhichao','EOS')
+try:
+    res = client.get_account('zhangzhichao')
+    print(res)
+except Exception as err:
+    print('fail')
+
```
Comment (EOSBIXIN) [inline eosapi/httpapi/README.md]: we should move the basic Client description into ../../README.md file, so others will see how to use the tool

File: eosapi/httpapi/http_client.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+    HttpAPIError,
+)
+from urllib3.connection import HTTPConnection
+from urllib3.exceptions import (
+    MaxRetryError,
+    ReadTimeoutError,
+    ProtocolError,
+)
+
+logger = logging.getLogger(__name__)
```
Comment (EOSBIXIN) [inline eosapi/httpapi/http_client.py]: we should the same logs/logger, avoid to defined duplicate log

File: startmonster.py [hunk 4]
```diff
@@ -24,7 +26,17 @@
 
 def check_exist_acct(conf_dict):
     '''check the common_params.exist_account availability'''
-    # TODO: need to be realized
+
+    # NOTE: get account that you need open http plugin.
+    # TODO add cli api to solve not open http plugin,
+    # TODO but by this way you need run this program in your nodeos server
```
Comment (EOSBIXIN) [inline startmonster.py]: maybe we can remove the TODO comments

**Review by EOSBIXIN — CHANGES_REQUESTED** [review]
thanks for the basic tool, adjust more pls.

**Comment by threadshare** [comment]
I have cleaned up the code and unified log, and modified readme(en), and I tested startmonster, it worked well.

**Pull request merged**